src/Workspaces/Core/Portable/Editing/ImportAdderService.cs

```diff
@@ -44,8 +44,8 @@ public enum Strategy
             var model = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
             Contract.ThrowIfNull(model);
             var root = await model.SyntaxTree.GetRootAsync(cancellationToken).ConfigureAwait(false);
-            var addImportsService = document.Project.LanguageServices.GetRequiredService<IAddImportsService>();
-            var generator = SyntaxGenerator.GetGenerator(document);
+            var addImportsService = document.GetRequiredLanguageService<IAddImportsService>();
+            var generator = document.GetRequiredLanguageService<SyntaxGenerator>();
```

Comment:
😕 Why did this line change?

Reply:
Just for consistency. I can revert if you feel otherwise.

Reply:
```suggestion
            var generator = SyntaxGenerator.GetGenerator(document);
```

Reply:
What is the advantage of using this static method over `GetRequiredLanguageService` given SyntaxGenerator is a language service? I am going to leave this as is for now, and we can make all the usages in our code bases consistent with a follow-up PR.

Reply:
Tracked as a follow-up cleanup in https://github.com/dotnet/roslyn/issues/41462